support-models/src/main/scala/com/gu/support/config/Stages.scala

```diff
@@ -18,7 +18,7 @@ object Stages {
 object Stage {
   def fromString(s: String): Option[Stage] = condOpt(s) {
     case "DEV" => DEV
-    case "CODE" => CODE
+    case "code" => CODE
```

Comment:
🤣

Reply:
Oops, I was looking out for that as well!